project/RELEASES.md

```diff
@@ -0,0 +1,168 @@
+# Scala LTS maintenance and release cycle
+
+## Scala LTS and Scala Next
+
+Since Scala 3.3.0, there are always two versions of Scala 3 being developed at the same time: Scala Next and Scala LTS.
+
+Scala Next continues all practices since before the 3.3.0. It maintains backward output (TASTy and binary) compatibility between all versions and forward output compatibility between patch releases of the same minor version. This version is being actively developed on the `main` branch. All new bug fixes and features will be merged to that version, i.e., to the `main` branch.
+
+Scala LTS is a selected minor version (currently 3.3) that, unlike all other minor versions, has patch releases even though there is a newer minor version. Aside from our standard output compatibility guarantees, it is intended to preserve backward source compatibility. As no new features and bugfixes (with the rare exception of LTS-only hotfixes) will be developed for LTS, but instead, selected changes from the Next branch will be backported, there is no development branch for Scala LTS. All changes are merged to release branches instead. The commit marked with a full release tag for any version is the base for the release branch for the next version.
+
+## What should be backported?
+
+The decision of what changes merged to the `main` branch should be backported is taken according to the following flowchart:
+
+```mermaid
+flowchart TB
+    start(["PR relevant for LTS was merged to main branch"])
+    -->
+    cve{"Is this
+         a fix for a CVE?"}
```

Comment:
Might be a good idea to spell out these acronyms at least once.

Reply:
"CVE" can be found online if you do not know, but "CB" is harder to understand. I assumed that CB stands for community build, is that correct?